### PR TITLE
Add events to GraphQL endpoint

### PR DESCRIPTION
**Pull request by ivankocienski**
You can now query the API with things like this for event index:
```
query {
    allEvents {
        id
        summary
        description
    }
}
```

And this for show event:
```
query {
    event(id:123) {
        id
        summary
        description
        address {
            streetAddress
            city
            postcode
        }
    }
}
```

### CODE REVIEW

File: app/graphql/types/address_type.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module Types
+  class AddressType < Types::BaseObject
+
+    description 'A partner'
```
Comment (kimadactyl) [inline app/graphql/types/address_type.rb:4]: An address?

File: app/graphql/types/address_type.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+    # field :id, ID, null: false
+    field :street_address, String
+    field :street_address2, String
+    field :street_address3, String
```
Comment (kimadactyl) [inline app/graphql/types/address_type.rb:9]: could we also give people access to `address.full_street_address` and `address.all_address_lines` from the model just for convenience?

File: app/graphql/types/query_type.rb [hunk 3]
```diff
@@ -23,6 +23,24 @@ def all_partners
...
 
+    ####
+
+    field :event, EventType, "Find Event by ID" do
+      argument :id, ID
+    end
+
+    field :all_events, [EventType]
+
+    def event(id:)
```
Comment (kimadactyl) [inline app/graphql/types/query_type.rb:34]: is this trailing colon correct? just looks odd :)

Comment (ivankocienski) [reply app/graphql/types/query_type.rb:34]: this i believe is a fancy ruby convenience way of doing `def event(opts={})` followed by `Event.find(opts[:id])`

**Pull request merged**